preliz/internal/optimization.py

```diff
@@ -186,6 +186,32 @@ def negll(params, dist, sample):
     return opt
 
 
+def optimize_dirichlet_mode(lower_bounds, mode, tau, new_prob, target_mass, _dist, alpha):
```

Comment:
This is missing the docstring, that is present on `dirichlet_prior_demo.py`. Move the relevant information here and deleted `dirichlet_prior_demo.py`.

Reply:
this is about the prob_approx function, right? ordo you mean something else?

Reply:
Ohh no, I just misread the file